js/common/lib/env-impl.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {Env} from './env';
+
+export class EnvIml implements Env {
+  // TODO standadize the getter and setter convention in env for other fields.
+  set logLevel(value: 'verbose'|'info'|'warning'|'error'|'fatal'|undefined) {
```

Comment:
if we use `set logLevel(value:unknown)`, does typescript complains about it incompatible with interface Env? looks like the implementation did type check